packages/logger/src/Logger.ts

```diff
@@ -151,7 +152,7 @@ class Logger extends Utility implements ClassThatLogs {
    */
   public constructor(options: ConstructorOptions = {}) {
     super();
-
+    this.initOptions = options;
```

Comment:
I would move this inside `setOptions`

Reply:
You are right. I'll created a method like `setInitOptions(options)`, initialize `initOptions` inside it, and put it in `setOptions` like other things. This method will be helpful, to update `initOptions` with attributes that can be added to an existing Logger instance.